cores/esp8266/pgmspace.h

```diff
@@ -39,6 +41,8 @@ typedef uint16_t prog_uint16_t;
 typedef int32_t prog_int32_t;
 typedef uint32_t prog_uint32_t;
 
+#endif // defined(__PROG_TYPES_COMPAT__)
```

Comment:
@igrr now that the should-have-been-deprecated typedefs are using this avr macro, is it OK to keep it for pgm_read_* type cast too ?

Reply:
Yes, should be okay.